src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -78,6 +91,8 @@ public Stage getPrimaryStage() {
 
     private void setAccelerators() {
         setAccelerator(helpMenuItem, KeyCombination.valueOf("F1"));
+        // breaks the program somehow???
+        // setAccelerator(toggleConcertContactVisibilityItem, KeyCombination.valueOf("Ctrl+Tab"));
```

Comment:
any1 know y

Reply:
did you try setting different KeyCombinations instead of ctrl+tab like ctrl+<some other key>

Reply:
Yeah Ctrl+Tab also worked if i set it for the Help function. But it crashes for the show cc for some reason no matter what i do with it. Disabling the help function also didnt work. Without setting the key combi the show cc looks like it works perfectly fine tho.

Reply:
toggleConcertContactVisibilityItem -> toggleConcertContactViewItem